### PR TITLE
Enable mouse only if it was prev enabled and based on scr.wheel

### PR DESCRIPTION
**Pull request by deepakchethan**

### CODE REVIEW

File: libr/core/visual.c [hunk 1]
```diff
@@ -3247,7 +3252,8 @@ R_API int r_core_visual_cmd(RCore *core, const char *arg) {
 				}
 			}
 			break;
-		case '/':
+		case '/': {
+			bool mouse_state = r_cons_singleton ()->mouse;
```
Comment (radare) [inline libr/core/visual.c]: i would make this into a separate static function:

static bool __holdMouseState(RCore *core) {
    int m = r_cons_singleton()->mouse;
	r_cons_enable_mouse (false);
       return m;
}

static void __releaseMouseState(RCore *core, bool oldState) {
  r_cons_enable_mouse (oldState && r_config_get_i (core->config, "scr.wheel"));
}

this way the repeated logic is in a separate function ,written only once. and have a meaningful name to please the reader

Comment (deepakchethan) [reply libr/core/visual.c]: Yeah, will change it in some time

**Comment by radare** [comment]
ping

**Comment by deepakchethan** [comment]
> ping

Ping reply, Made the changes :)

**Pull request merged**